src/v/cluster/feature_manager.cc

```diff
@@ -274,32 +276,38 @@ feature_manager::report_enterprise_features() const {
     return report;
 }
 
-ss::future<> feature_manager::maybe_log_license_check_info() {
-    auto license_check_retry = std::chrono::seconds(60 * 5);
+ss::future<> feature_manager::maybe_log_periodic_reminders() {
+    auto reminder_period = std::chrono::seconds(60 * 5);
     auto interval_override = std::getenv(
       "__REDPANDA_LICENSE_CHECK_INTERVAL_SEC");
     if (interval_override != nullptr) {
         try {
-            license_check_retry = std::min(
-              std::chrono::seconds{license_check_retry},
+            reminder_period = std::min(
+              std::chrono::seconds{reminder_period},
               std::chrono::seconds{std::stoi(interval_override)});
             vlog(
               clusterlog.info,
-              "Overriding default license log annoy interval to: {}s",
-              license_check_retry.count());
+              "Overriding default reminder period interval to: {}s",
```

Comment:
It's probably worth extracting out the duplicate `_license_nag_is_set` functions.

Reply:
Oh I missed changing those as well